files/zh-cn/web/javascript/guide/iterators_and_generators/index.md

```diff
@@ -123,7 +161,7 @@ for (let value of myIterable) {
 
 ### 用于可迭代对象的语法
 
-一些语句和表达式专用于可迭代对象，例如 {{jsxref("Statements/for...of","for-of")}} 循环，{{jsxref("Operators/Spread_operator","展开语法")}}，{{jsxref("Operators/yield*", "yield*")}} 和 {{jsxref("Operators/Destructuring_assignment", "解构赋值")}}。
+一些语句和表达式专用于可迭代对象，例如 {{jsxref("Statements/for...of", "for...of")}} 循环，{{jsxref("Operators/Spread_syntax", "展开语法")}}，{{jsxref("Operators/yield*", "yield*")}} 和 {{jsxref("Operators/Destructuring_assignment", "解构赋值")}}。
```

Comment:
Could you update the en-US document to reflect the following translation?

```suggestion
一些语句和表达式专用于可迭代对象，例如 {{jsxref("Statements/for...of", "for...of")}} 循环、{{jsxref("Operators/Spread_syntax", "展开语法", "", 1)}}、{{jsxref("Operators/yield*", "yield*")}} 和{{jsxref("Operators/Destructuring_assignment", "解构赋值", "", 1)}}。
```

Reply:
https://github.com/mdn/content/pull/30054